stream-video-android-ui-compose/src/main/kotlin/io/getstream/video/android/compose/ui/components/video/config/VideoRendererConfig.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2014-2024 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-video-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.video.android.compose.ui.components.video.config
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Immutable
+import androidx.compose.ui.Modifier
+import io.getstream.video.android.compose.ui.components.video.DefaultMediaTrackFallbackContent
+import io.getstream.video.android.compose.ui.components.video.VideoScalingType
+import io.getstream.video.android.core.Call
+
+@Immutable
+public data class VideoRendererConfig(
+    var mirrorStream: Boolean = false,
+    val scalingType: VideoScalingType = VideoScalingType.SCALE_ASPECT_FILL,
+    val fallbackContent: @Composable (Call) -> Unit = {},
```

Comment:
Do we need to make `mirrorStream` as mutable? If so, we should remove the `@Immutable` annotation.

Reply:
It shouldn't be a `var`. I will change it to `val` and keep the annotation.